core/src/banking_stage/leader_slot_metrics.rs

```diff
@@ -52,6 +54,53 @@ pub(crate) struct ProcessTransactionsSummary {
 
     // Breakdown of all the transaction errors from transactions passed for execution
     pub error_counters: TransactionErrorMetrics,
+
+    pub min_prioritization_fees: usize,
+    pub max_prioritization_fees: usize,
+}
+
+// Metrics describing prioritization fee information for each transaction storage before processing transactions
+#[derive(Debug, Default)]
+struct LeaderPrioritizationFeesMetrics {
+    // minimum prioritization fees in the MinMaxHeap
+    min_prioritization_fees_per_cu: u64,
+    // maximum prioritization fees in the MinMaxHeap
+    max_prioritization_fees_per_cu: u64,
+}
+
+impl LeaderPrioritizationFeesMetrics {
+    fn new(unprocessed_transaction_storage: Option<&UnprocessedTransactionStorage>) -> Self {
+        if let Some(unprocessed_transaction_storage) = unprocessed_transaction_storage {
+            Self {
+                min_prioritization_fees_per_cu: unprocessed_transaction_storage
+                    .get_min_priority()
+                    .unwrap_or_default(),
+                max_prioritization_fees_per_cu: unprocessed_transaction_storage
+                    .get_max_priority()
+                    .unwrap_or_default(),
+            }
+        } else {
+            Self::default()
+        }
+    }
+
+    fn report(&self, id: u32, slot: Slot) {
+        datapoint_info!(
+            "banking_stage-leader_prioritization_fees_info",
```

Comment:
Agree needing a way to determine how priority is honored by leaders. I was naively thinking to track last landed transaction's priority fee cross threads and cross accounts, if any new transaction has higher priority than last landed tx (per account), then one violation is booked. But this is cumbersome. How are you planning to use min/max?

Reply:
We have created and working on the dashboard, it tracks transactions with prioritization fees and show min, max etc for all the account. https://bankingstage-dashboard.mngo.cloud/recent-blocks
With this metrics we can more explain what each worker was looking before choosing the transactions.

Reply:
the dashboard is kool!